TaskEstimation.md

```diff
@@ -0,0 +1,15 @@
+Task Estimation in Scrum
+
+Key Concepts:
+1.Sprint planning:
+This is a crucial part in a scrum meeting as this is where the team collaberates and determins whcich tasks can be done in the current sprint. The estimation is based off the resources and time that they have for the sprint. This practice would highlight communication and teamwork skills. According to wrike.com( https://www.wrike.com/scrum-guide/scrum-sprint-planning/) it states that the main importance of  sprint planning is "Sets clear defined goals, creates a communication platform and collaberation".
+
+2.User Stories:
+User Stories provides an informal, natural language description of a feature of the software or product from the end-user perspective. According to this article user stories are very important because it "Improves transparency and Highest value is delivered"(https://www.simplilearn.com/tutorials/agile-scrum-tutorial/user-stories) without user stories there could be confusion amongst the team as there wont be anything with a clear and cut description of the tasks.
+
+
+Challenges and Skills:
+1.Effort of peers: If certain members within the team are not "pulling their weight" it could affect the group as a whole and can lead to several disagreements and could affect the team as a whole. In this article it briefly describes a scenario but the key thing is from here that the slack of one person was creating frustration "Frustrated, he looked to Ann's desk and she was not there. Again"(https://medium.com/serious-scrum/scrum-team-interactions-under-pressure-ac242e7ec110).
```

Comment:
could elaborate more as to why ann was frustrated in regards to effort of peers.